dev/com.ibm.ws.jca.utils/resources/com/ibm/ws/jca/utils/internal/resources/JcaUtilsMessages.nlsprops

```diff
@@ -191,5 +191,13 @@ J2CA9942.cfgprop.invalid.parameters.explanation=The annotation @ConfigProperty m
 J2CA9942.cfgprop.invalid.parameters.useraction=Investigate the resource adapter vendor's documentation and problem determination processes.
 
 J2CA9943.missing.connector.dd=J2CA9943E: The resource adapter {0} is invalid. It should contain either a deployment descriptor or a JavaBean class annotated with one of the resource adapter annotations.
-J2CA9943.missing.connector.dd.explanation=A valid resource adapter must have either a deployment descriptor or a JavaBean class annotated with one of @Connector, @AdministeredObject, @ConnectionDefinition. @ConnectionDefinitions or @Activation annotations.
+J2CA9943.missing.connector.dd.explanation=A valid resource adapter must have either a deployment descriptor or a JavaBean class annotated with one of @Connector, @AdministeredObject, @ConnectionDefinition, @ConnectionDefinitions or @Activation annotations.
 J2CA9943.missing.connector.dd.useraction=Investigate the resource adapter vendor's documentation and problem determination processes.
+
+J2CA9944.missing.connector.jakarta=J2CA9944E: The resource adapter {0} should contain either a deployment descriptor or a JavaBean class annotated with one of the resource adapter annotations from the Jakarta Connectors specification. If the resource adapter uses classes from the Java EE Connector Architecture 1.7 or lower version specification (javax.resource.spi package), then you should be using the jca-1.7 or lower version feature rather than Jakarta EE features.
+J2CA9944.missing.connector.jakarta.explanation=The resource adapter must have either a deployment descriptor or a JavaBean class annotated with one of @Connector, @AdministeredObject, @ConnectionDefinition, @ConnectionDefinitions or @Activation annotations from the jakarta.resource.spi package.
+J2CA9944.missing.connector.jakarta.useraction=Use the jca-1.7 or lower version feature with a resource adapter that implements the Java EE Connector Architecture specification.
+
+J2CA9945.missing.connector.javax=J2CA9945E: The resource adapter {0} should contain either a deployment descriptor or a JavaBean class annotated with one of the resource adapter annotations from the Java EE Connector Architecture specification. If the resource adapter uses classes from the Jakarta Connectors 2.0 or higher version specification (jakarta.resource.spi package), then you should be using the connectors-2.0 or higher version feature rather than Java EE features.
```

Comment:
suggested edit:

The {0} resource adapter can contain either a deployment descriptor or a JavaBeans class that  is annotated with one of the resource adapter annotations from the Java EE Connector Architecture specification. If the resource adapter uses classes from the Jakarta Connectors 2.0 or higher version specification (jakarta.resource.spi package), use the connectors-2.0 or higher version feature rather than Java EE features.